docs/intercoder_reliability.md

```diff
@@ -0,0 +1,117 @@
+# Results from corpus tagging
+
+
+Each user tagged 100 documents (unless otherwise specified). Each document were tagged
+on a per "sentence" basis. A "sentence" is defined by a newline split. "sentences", longer
+that 1000 characters were split into multiple sentences of 1000 characters or less.
+This tagging scheme similar to 
+([Kreutzer et al., 2022](https://direct.mit.edu/tacl/article/doi/10.1162/tacl_a_00447/109285/Quality-at-a-Glance-An-Audit-of-Web-Crawled)).
+Each tag where put into one of the following categories:
```

Comment:
```suggestion
Each user tagged 100 documents unless otherwise specified. Documents were split by newlines into text-blocks, block was rated. 

Text-blocks longer than 1000 characters were split into multiple blocks of 1000 characters or less.

This tagging scheme is similar to 
([Kreutzer et al., 2022](https://direct.mit.edu/tacl/article/doi/10.1162/tacl_a_00447/109285/Quality-at-a-Glance-An-Audit-of-Web-Crawled)).

Each block was put into one of the following categories:
```